apps/hash-graph/bin/cli/Cargo.toml

```diff
@@ -14,6 +14,7 @@ axum = "0.6.3"
 clap = { version = "4.1.4", features = ["cargo", "derive", "env", "wrap_help"] }
 clap_complete = "4.1.1"
 error-stack = { version = "0.3.0", features = ["spantrace"] }
+futures = "0.3.25"
```

Comment:
Do we need this dependency? If this isn't optional and enabled by `type-fetcher`, I dont' think we'd need this

Reply:
Made this optional in https://github.com/hashintel/hash/pull/2038